### PR TITLE
Update bedtools to 2.29.0

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by mvdbeek** [comment]
https://travis-ci.org/galaxyproject/tools-iuc/jobs/528792657#L2619 is due to https://github.com/arq5x/bedtools2/issues/727 which is fixed on the master branch but no release yet.

**Comment by mvdbeek** [comment]
Hmm, there's still a couple of bugs in 2.29 that prevent an update:

- bamtofastq segfaults, but we could deprecate it, samtools fastq is a better alternative (also mentioned in the help text of the tool)
- bedtools_bedpetobam produces truncated and unusable bam files

I'd be in favor of keeping 2.27.1 for bedtools_bedpetobam and upgrade the rest.

**Comment by mvdbeek** [comment]
Green! Finally :)

File: tools/bedtools/bedpeToBam.xml [hunk 1]
```diff
@@ -3,7 +3,10 @@
     <macros>
```
Comment (nsoranzo) [inline tools/bedtools/bedpeToBam.xml]: I think you need to change the tool `version` to `2.27.1` .

**Pull request merged**